features/search_profiles_by_distance.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Search profiles by distance
+
+Background:
+	Given a person is registered
+	And that they have signed in
```

Comment:
I suggest indenting the `And` keywords so that you can see the `Given`, `When` and `Thens`